### PR TITLE
docs: Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by zyxiaoyuer**

### CODE REVIEW

**Review by jayzhang — APPROVED** [review]

File: packages/vscode-extension/WHATISNEW.md [hunk 1]
```diff
@@ -10,6 +11,20 @@
 - [2.8.0 - Oct 18 2021](#280---oct-18-2021)
 - [2.7.0 - Sep 17 2021](#270---sep-17-2021)
 
+# Changelog
+
+## 3.6.0 - Mar 21, 2022
+Incremental version for Teams Toolkit with multiple bugs fixes and the following updates:
+
+New Features:
+- User can use Teams Toolkit to create CI/CD pipelines for GitHub, Azure DevOps and Jenkins.
```
Comment (IvanJobs) [inline packages/vscode-extension/WHATISNEW.md:20]: Align with the following tooltip would be better.
`Create workflow automation templates for Github, Azure DevOps and Jenkins`
We're not going to help users create pipeline, just workflow/pipeline templates in yml/jenkins files.

Comment (IvanJobs) [inline packages/vscode-extension/WHATISNEW.md:20]: Align with the following tooltip would be better.
`Create workflow automation templates for Github, Azure DevOps and Jenkins`
We're not going to help users create pipeline, just workflow/pipeline templates in yml/jenkins files.

File: packages/vscode-extension/WHATISNEW.md [hunk 2]
```diff
@@ -10,6 +11,20 @@
...
+
+	![cicd workflow](https://raw.githubusercontent.com/OfficeDev/TeamsFx/main/packages/vscode-extension/img/cicd_workflow.png)
+
+Enhancement:
```
Comment (IvanJobs) [inline packages/vscode-extension/WHATISNEW.md:24]: Enhancement(s)?

Comment (IvanJobs) [inline packages/vscode-extension/WHATISNEW.md:24]: Enhancement(s)?

**Review by HuihuiWu-Microsoft — APPROVED** [review]
LGTM

**Review by tecton — DISMISSED** [review]

**Review by eriolchan — APPROVED** [review]

**Pull request merged**